.formatter.exs

```diff
@@ -1,4 +1,6 @@
 # Used by "mix format"
 [
-  inputs: ["{mix,.formatter}.exs", "{config,lib,test}/**/*.{ex,exs}"]
+  inputs:
+    ["{mix,.formatter}.exs", "{config,lib}/**/*.{ex,exs}"] ++
+      ((Path.wildcard("test/*") -- ["test/generated"]) |> Enum.map(&(&1 <> "/**/*.{ex,exs}")))
```

Comment:
this does not bring back the regression about local formatting taking forever, right?